src/Common/tests/System.Net/Sockets/Performance/SocketPerformanceTests.cs

```diff
@@ -8,7 +8,7 @@
 using Xunit;
 using Xunit.Abstractions;
 
-namespace System.Net.Sockets.Performance.Tests
+namespace System.Net.Sockets.Tests
```

Comment:
That doesn't really work for us as we want clear separation between the test types.